setup.py

```diff
@@ -89,68 +91,30 @@ def recursive_data_files(directory, install_directory):
         version = match.group(1)
 
 
-# If on Windows, construct an exe distribution
+# snapcraftctl is not in console_scripts because we need a clean environment.
+# but do not include if building on Windows.
```

Comment:
this guard might be useful for mac as well

Reply:
ah, nevermind!

Reply:
Will update the comment.

Reply:
Updated to only set it for linux.